mne/io/base.py

```diff
@@ -1647,8 +1647,9 @@ def save(self, fname, picks=None, tmin=0, tmax=None, buffer_size_sec=None,
             can be loaded with the MNE command-line tools. See raw.orig_format
             to determine the format the original data were stored in.
         overwrite : bool
-            If True, the destination file (if it exists) will be overwritten.
-            If False (default), an error will be raised if the file exists.
+            If True, the destination file (if it exists) will be overwritten,
+            data must be preloaded to overwrite. If False (default), an error
```

Comment:
This sounds a bit misleading -- data must be preloaded to overwrite *only if writing to the same file name*. It's probably worth clarifying this somehow (I don't really like my sentence either) and making it a separate sentence.

Reply:
I agree, I'd add another sentence after the two original ones.